pkg/kopia/snapshot/snapshot.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/kanisterio/kanister/pkg/kopia"
-	"github.com/kanisterio/kanister/pkg/kopia/repository"
+	reposerverclient "github.com/kanisterio/kanister/pkg/kopia/repository/client"
```

Comment:
Couldn't comment on the above file change. I would just create `repository/client.go` to simplify things

```suggestion
	repoclient "github.com/kanisterio/kanister/pkg/kopia/repository/client"
```